### PR TITLE
Fixed alignment

### PR DESCRIPTION
**Pull request by sharpninja**
Removes BOM on load, adds it back on save.

### CODE REVIEW

File: src/QuickPad.UI/QuickPad.UI/MultilingualResources/QuickPad.UI.de.xlf [hunk 1]
```diff
@@ -1,885 +1,885 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (yaichenbaum) [inline src/QuickPad.UI/QuickPad.UI/MultilingualResources/QuickPad.UI.de.xlf:1]: Can you revert the changes to this file?

File: src/QuickPad.UI/QuickPad.UI/Strings/de/Resources.resw [hunk 2]
```diff
@@ -1,711 +1,711 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (yaichenbaum) [inline src/QuickPad.UI/QuickPad.UI/Strings/de/Resources.resw:1]: Can you revert the changes to this file?

**Comment by diligamer** [comment]
@sharpninja Please respond To @yaichenbaum

**Comment by yaichenbaum** [comment]
@diligamer Sharp Ninja is aware of my comment and will address it when he gets a chance.
